cloudfomation.yaml

```diff
@@ -21,13 +21,69 @@ Parameters:
     Type: String
     MinLength: 1
     Description: Name of CodePipeline pipeline
-
-
+  EncryptionAtRest:
+    Type: String
+    Default: "No"
```

Comment:
lets make default Yes as default we wish to keep data encrypted

Reply:
Done. Thank you!